space-framework/go.mod

```diff
@@ -3,104 +3,74 @@ module github.com/kubestellar/kubestellar/space-framework
 go 1.19
 
 require (
-	github.com/go-logr/logr v1.2.3
+	github.com/go-logr/logr v1.2.4
 	github.com/kcp-dev/apimachinery/v2 v2.0.0-alpha.0
-	github.com/kcp-dev/client-go v0.0.0-20230126185145-aeff170a288b
-	github.com/kcp-dev/kcp v0.11.0
 	github.com/kcp-dev/kcp/pkg/apis v0.11.0
+	github.com/kcp-dev/kcp/pkg/client v0.0.0-20230328080949-0f3bb69fdc08
```

Comment:
Is this the right commit to use?  https://github.com/kcp-dev/kcp/commit/0f3bb69fdc08 is much later than v0.11.0 .

Reply:
I didn't touch this. I think git mod tidy put it automatically.  Which version should we use ? In any case this is for the KCP provider that currently is not going to be used. I'll try to match to the 0.11